packages/nextra/src/plugin.ts

```diff
@@ -57,13 +59,25 @@ export async function collectFiles(
   dir: string,
   locales = DEFAULT_LOCALES,
   route = '/',
-  fileMap: FileMap = Object.create(null)
+  fileMap: FileMap = Object.create(null),
+  isFollowingSymlink = false
```

Comment:
this function now accepts too many arguments, should be refactored to object in the future :)

Reply:
agree, I wasn't sure about including bigger refactoring in the scope of this work. happy to open a following pr with a refactored signature